ibc-core/ics02-client/context/src/consensus_state.rs

```diff
@@ -18,5 +19,5 @@ pub trait ConsensusState: Send + Sync {
     /// Serializes the `ConsensusState`. This is expected to be implemented as
     /// first converting to the raw type (i.e. the protobuf definition), and then
     /// serializing that.
-    fn encode_vec(self) -> Vec<u8>;
+    fn into_any(self) -> Any;
```

Comment:
- The doc string needs to be updated.
- Also, this may be removed with a trait bound `Into<Any>`.